Mathlib/Data/Fintype/Pi.lean

```diff
@@ -77,6 +84,25 @@ theorem piFinset_disjoint_of_disjoint (t₁ t₂ : ∀ a, Finset (δ a)) {a : α
       (congr_fun eq₁₂ a)
 #align fintype.pi_finset_disjoint_of_disjoint Fintype.piFinset_disjoint_of_disjoint
 
+lemma piFinset_image [∀ a, DecidableEq (δ a)] (f : ∀ a, γ a → δ a) (s : ∀ a, Finset (γ a)) :
+    piFinset (fun a ↦ (s a).image (f a)) = (piFinset s).image fun b a ↦ f _ (b a) := by
+  ext; simp only [mem_piFinset, mem_image, Classical.skolem, forall_and, Function.funext_iff]
+
+lemma image_piFinset_eval (t : ∀ a, Finset (δ a)) (a : α) [DecidableEq (δ a)]
```

Comment:
Please add one of the `⊆` version without `Nonempty` assumption, see `Set.eval_image_pi_subset`.